src/components/creator/Editor/CreatorTryMode.tsx

```diff
@@ -14,10 +14,14 @@ const HeaderContent = (challenge: SerializedChallenge) => {
     const {t} = useTranslation('creator')
 
     return <>
+        <Stack direction="row" sx={{alignItems: 'center'}}>
```

Comment:
![imagen](https://github.com/Program-AR/pilas-bloques-react/assets/5421992/9c138ea5-ab07-4189-a006-2f9d02e0864a)

Reply:
Cual es la diferencia con usar la prop del componente directametne? Osea hacer
```ts
<Stack direction="row" alignItems="center">
```

Reply:
Antes que nada, **hay aún más formas**  de las que mencionás :laughing: 

- Acá tenés las básicas de React/JSX: https://css-tricks.com/different-ways-to-write-css-in-react/ (están bastante bien explicadas y ejemplificadas).
- Y después yo ahí metí `sx` y vos mencionaste `alignItems`, que son properties de la biblioteca **MUI**.
https://mui.com/material-ui/customization/how-to-customize/

Una guía para leer todo eso (y para tomar decisiones nosotros) es:
- ¿Vas a **reusar** estilos?
- ¿Vas a **usar mediaqueries**? ¡MUI tiene queries ya resueltas! ¿Vas a usar las que define **por defecto MUI**?
- ¿Vas a **cambiar dinámicamente con un if** los estilos?
- ¿Vas a darle estilos a un JSX plano **o a un componente MUI**?

La razón por la que es confuso es que el 80% de las veces las respuestas son todas no, y en ese caso **todas las estrategias en general dan lo mismo** :laughing: .  O sea, no dan lo mismo, podría diferir el comportamiento, pero en general podemos decir que dan lo mismo y usar una o la otro. Lo que cambia es la forma de renderizarse al html, [como explican acá](https://stackoverflow.com/questions/72527461/when-should-i-use-style-instead-of-sx-prop-in-material-ui) al comparar `sx` con `style`.

Pero en general:
- Antes que nada, como yo no sé demasiado de esto y estoy también aprendiendo, no me animo a definir nada a rajatabla, y les escucho. Ahora salgo a caminar con Gise y le pregunto qué opina :P.
- Estoy tentado a considerar equivalente lo que propusiste (usar la prop `alignItems`) especialmente si da lo mismo.
- Si tu código JSX se empieza a llenar de styling es síntoma de que como mínimo querés extraer el estilo a un objeto json, o a un archivo .css, y usar las `className` para stylear, o el helper `styled`. Así separás la definición de estilos de la estructura de la vista. Justo en este archivo no hay absolutamente nada de estilos. (amo cuando los defaults de mui/de flex andan por sí solos)
- No dejemos cosas por las dudas. CSS es complejo y no ayuda dejar código al pedo. Dejemos lo que sabemos a ciencia cierta que hace algo.
- Dani para las media queries usó las de MUI, me parece acertado eso. No agregar "cortes" diferentes a los que ya tiene MUI.
- Lo que importa es establecer la teoría (**la estrategia**, como la que discutimos aquél día con la pantalla ppal). Y luego debuggear un poquito cuando no anda.